src/symboltable.c

```diff
@@ -375,6 +375,13 @@ void vmp_scan_profile(int fileno, int dump_nat_sym, void *all_code_uids)
     khash_t(ptr) * nat_syms = kh_init(ptr);
     khiter_t it;
 
+#ifndef RPYTHON_VMPROF
+    PyObject *all_done_uids = NULL;
+    if (all_code_uids != NULL) {
+        all_done_uids = PySet_New(NULL);
```

Comment:
Yeah, here's my confusion - we already use CPython API further down the line, similarly if an `#ifndef RPYTHON_VMPROF` block. :(

...so, is the right fix to change both?

Or is the other one special enough to be kept around?